content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -13,7 +13,7 @@ description: Creating Secret objects using resource configuration file.
 
 <!-- steps -->
 
-## Create the Secret {#create-the-config-file}
+## Create the Secret {#create-the-secret}
```

Comment:
In English document, we usually don't have to add `{#anchor}` for headers.
It's intentionally add the "inconsistent" link anchor to provide backward compatibility.

Users usually don't look at https://k8s.io/link#anchor (or allow some inconsistency), but it's not good to broken previous links in this repo and external sites.

Reply:
I just guess whether the author forgot to change this anchor. Then, change it back...